app/services/time_of_need_service.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+require 'restforce'
+require 'faraday'
+
+class TimeOfNeedService
+
+  @conn = Faraday.new(:url => 'https://va--mbmssit.cs33.my.salesforce.com/services/oauth2/token')
```

Comment:
We need to make sure we are not hardcoding urls, please use the Settings pattern already in place by the vets-api